### ISSUE TITLE
fix: dependencies#duplicate_classes

### ISSUE THREAD

**Issue by codeforce-dev** [thekorn/sourcepoint_unified_cmp#38]
**Description**

Thanks for this plugin. In Android I get an error when compiling. MultiDex ist enabled.

Flutter 3.19

```
Type FlutterError is defined multiple times: /Users/user/Documents/Development/Projects/apps/app/myapp/build/wakelock_plus/.transforms/58d63efe623f1987cc4c4869d8c74051/transformed/debug/FlutterError.dex, /Users/user/Documents/Development/Projects/apps/app/myapp/build/sourcepoint_unified_cmp_android/.transforms/ff8d3284dc667e5989f5bff6780960b7/transformed/debug/FlutterError.dex
```

```
* What went wrong:
Execution failed for task ':app:mergeLibDexDaDebug'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.DexMergingTaskDelegate
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.tasks.DexMergingWorkAction
         > com.android.builder.dexing.DexArchiveMergerException: Error while merging dex archives: 
           Type FlutterError is defined multiple times: /Users/user/Documents/Development/Projects/apps/app/myapp/build/wakelock_plus/.transforms/58d63efe623f1987cc4c4869d8c74051/transformed/debug/FlutterError.dex, /Users/user/Documents/Development/Projects/apps/app/myapp/build/sourcepoint_unified_cmp_android/.transforms/ff8d3284dc667e5989f5bff6780960b7/transformed/debug/FlutterError.dex
           Learn how to resolve the issue at https://developer.android.com/studio/build/dependencies#duplicate_classes.
```

**Comment by codeforce-dev** [comment]
Now I renamed FlutterError in FlutterError2 and it works.

**Comment by thekorn** [comment]
@codeforce-dev pigeon provides a way to define a custom error class to prevent such issues.
I'm using it in the attached branch, and will release a new version of the package later this evening

**Comment by thekorn** [comment]
@codeforce-dev  fix released as [0.1.9](https://pub.dev/packages/sourcepoint_unified_cmp/versions/0.1.9)
